bootstrap/bootstrap.ps1

```diff
@@ -2,6 +2,7 @@ param
 (
     [string]$RepositoryRoot,
     [string]$ToolsLocalPath,
+    [string]$CliLocalPath,
```

Comment:
This needs to be made optional. We should probably make all these optional and given them reasonable defaults.